SpatialGDK/Documentation/content/glossary.md

```diff
@@ -31,13 +31,13 @@ Actor handover (`handover`) is a GDK-specific `UPROPERTY` tag. It allows games b
 
 ### Authority
 
-A [server-worker instance](#server-workers) has authority over an Actor if it has authority over the equivalent [entity](#entity)’s [SpatialOS component] `Position`. Having authority means it can make changes to the Actor by sending updates to the [SpatialOS Runtime](#spatialos-runtime). 
+If a [server-worker instance](#server-workers) has authority over an Actor, the server-worker instance can make changes to the Actor by sending updates to the [SpatialOS Runtime](#spatialos-runtime). A server-worker instance has authority over an Actor if it has authority over the equivalent [entity](#entity)’s [SpatialOS component](#spatialos-component) `Position`. 
```

Comment:
```suggestion
When a [server-worker instance](#server-workers) has authority over an Actor, the server-worker instance can make changes to the Actor by sending updates to the [SpatialOS Runtime](#spatialos-runtime). A server-worker instance has authority over an Actor if it has authority over the equivalent [entity](#entity)’s [SpatialOS component](#spatialos-component) `Position`. 
```
When anchors the statement more. (Not significant change but makes it a bit of a stronger assertion.)